academy/4-my-own-chain/game-forfeit.md

```diff
@@ -25,12 +25,12 @@ In the [previous section](./game-winner.md) you prepared expiration of games:
 
 ## New information
 
-An expired game will expire in two different cases:
+An expired game occurs in two different cases:
 
-1. It was never really played on so it is removed quietly.
-2. It was played on, making it a proper game, and forfeit is the outcome because a player failed to play in time.
+1. No moves were ever played, so it is removed quietly.
+2. One or more moves were played, making it a proper game, and the outcome is a forfeit because one player failed to play in time.
```

Comment:
```suggestion
1. Not more than a single move was ever played, so at least a player failed to express their interest in the game. The game is removed quietly.
2. Two or more moves were played, making it a proper game, and the outcome is a forfeit because one player failed to play in time.
```
@AndrewB9lab I am changing this to reflect the reality of the code.